packages/web/pages/user/my-account/curate-profile.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Protected } from '../../../components/Authorization';
+import { UserProfile } from '../../../components/UserProfile';
+import { CurateSpecialties, CurateFormSpecialties } from '../../../components/CurateSpecialties';
+import HeaderProfile from '../../../components/HeaderProfile';
+import { ROLES as rolesEnum } from '../../../utils/enums/api.enum';
+import { getReviewerUser } from '../../../services/user';
+
+const CurateProfile = ({ categories = [] }) => {
+	const normalizeCategories = categories
+		.map(
+			(category) =>
+				category.parent_id && {
+					category: {
+						label: categories.filter((c) => c.id === category.parent_id)[0].term,
+						value: categories
+							.filter((c) => c.id === category.parent_id)[0]
+							.id.toString(),
+					},
+					subCategory: {
+						label: category.term,
+						value: category.id.toString(),
+					},
+				},
+		)
+		.filter((category) => category);
+
+	return (
+		<Container>
+			<Protected userRole={rolesEnum.REVIEWER}>
+				<UserProfile />
+				<MainContentContainer>
+					<HeaderProfile />
+					<Grid>
+						<Sidebar>
+							<CurateFormSpecialties defaultValues={normalizeCategories} />
+						</Sidebar>
+						<CurateSpecialties data={categories} />
+					</Grid>
+				</MainContentContainer>
+			</Protected>
+		</Container>
+	);
+};
+
+CurateProfile.getInitialProps = async () => {
+	const {
+		data: { categories },
+	} = (await getReviewerUser()) || {};
```

Comment:
```suggestion
	const {
		data: { categories },
	} = (await getReviewerUser()) || {};
```

pq precisou desse fallback `|| {}` ? 🤔

Reply:
Removi o fallback 😄